client/components/charts/charts.directives.js

```diff
@@ -1216,9 +1216,11 @@ app.directive('heartbeatChart', ['$timeout', 'Report', '$routeParams', 'COLOURS'
     // ClientDetails.client.ap_mac = undefined;
 
     controller.$scope.$on('resizeClientChart', function (evt, type){
-      if (a) {
-        drawChart();
-      }
+      drawChart();
+    });
+
+    controller.$scope.$on('loadClientChart', function  (){
```

Comment:
We have removed this on the other charts now. Should not use on a general basis other than resize.